model/src/w3sdb1md.F90

```diff
@@ -231,12 +232,9 @@ SUBROUTINE W3SDB1 (IX, A, DEPTH, EMEAN, FMEAN, WNMEAN, CG, LBREAK, S, D )
 #endif
     !
     ! 0.  Initialzations ------------------------------------------------- /
-    !     Never touch this 4 lines below ... otherwise my exceptionhandling will not work.
```

Comment:
I was doing some testing below - and saw this "Never touch these four lines" comment.  I'm assuming @aronroland  that the exception handling still works.... but just thought I'd draw attention to this comment.

Reply:
Hi @JessicaMeixner-NOAA sorry for my late reply, yes this was a leftover from long ago so I cleaned the code thanks for looking at this.

Reply:
Hi @JessicaMeixner-NOAA, back to your comment on the UFS and the DB1 issue. I have check more the code and the reason is in this case not DB1 but rather my changes in w3wave, where we changed how the source terms are called in the implicit schemes for the sake of neumann boundary but also wetting and drying. I will expand a bit the text so it becomes more clear to everybody.

Reply:
Thanks for replying @aronroland  - and thanks for cleaning this up and all the bug fixes!